cli/script/command-executor.ts

```diff
@@ -1038,6 +1038,13 @@ export var release = (command: cli.IReleaseCommand): Promise<void> => {
                 .then((): void => {
                     log("Successfully released an update containing the \"" + command.package + "\" " + (isSingleFilePackage ? "file" : "directory") + " to the \"" + command.deploymentName + "\" deployment of the \"" + command.appName + "\" app.");
                 })
+                .catch((err: Error): void => {
+                    if (err.message === "Not Found") {
+                        err.message = "Release failed, the size of the upload may be too large.";
```

Comment:
Should we be more specific here and say that app's must be below 200mb in size? Also, the "Release failed," probably isn't necessary since it would already be clear that an error happened as part of the release.